honeybee_radiance/sensorgrid.py

```diff
@@ -190,6 +198,40 @@ def count(self):
         """Number of sensors."""
         return len(self._sensors)
 
+    @property
+    def room_identifier(self):
+        """Get or set text for the Room identifier to which this SensorGrid belongs.
+        
+        This will be used in the model_folder_dict method to narrow down the
+        number of aperture groups that have to be run with this sensor grid.
+        If None, the grid will be run with all aperture groups in the model.
+        """
+        return self._room_identifier
+
+    @room_identifier.setter
+    def room_identifier(self, n):
+        self._room_identifier = typing.valid_string(n)
+
+    def model_folder_dict(self, model=None):
```

Comment:
I would like some more generic name link `info_dict`. This information can be used in other cases more than just writing to a folder.

Reply:
Sounds good. `info_dict` it is.